package.json

```diff
@@ -1,8 +1,35 @@
 {
   "name": "ethereumjs-vm",
   "version": "2.3.3",
-  "description": "an ethereum VM implementation",
-  "main": "index.js",
+  "description": "An Ethereum VM implementation",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "coverage": "istanbul cover ./tests/tester.js -- -s",
+    "coveralls": "npm run coverage && coveralls <coverage/lcov.info",
+    "testVM": "node ./tests/tester -v",
+    "testState": "npm run build:dist && node ./tests/tester -s --dist",
+    "testBuildIntegrity": "npm run build:dist && node ./tests/tester -s --dist --test='stackOverflow'",
+    "testBlockchain": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --dist --excludeDir='GeneralStateTests'",
+    "testBlockchainGeneralStateTests": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --dist --dir='GeneralStateTests'",
+    "testBlockchainBlockGasLimit": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --dist --dir='bcBlockGasLimitTest'",
+    "testBlockchainValid": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --dist --dir='bcValidBlockTest'",
+    "testBlockchainTotalDifficulty": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --dist --dir='bcTotalDifficultyTest'",
+    "test": "node ./tests/tester -a",
+    "lint": "standard",
```

Comment:
Will `lint` ignore the `dist` directory?

Reply:
Yes.

Reply:
Yeah I see the `dist/**` line in a`package.json`, just the change view on github didn't made it apparent.